dtfs-central-api/src/errors/invalid-state-machine-transition.error.ts

```diff
@@ -12,7 +12,7 @@ type EntityParams = RequiredParams & {
 };
 
 export class InvalidStateMachineTransitionError extends ApiError {
-  private constructor(message: string) {
+  public constructor(message: string) {
```

Comment:
What is the reason we had to make this public?

Reply:
I did it at the time in order to remove the need for an eslint rule disable, but this was fixed as part of the prettier PR (I think) so I can set this back to `private`